### ISSUE TITLE
TypeError: Arguments to CompositeDisposable.add must have a .dispose() method

### ISSUE THREAD

**Issue by garettmd** [mupchrch/split-diff#72]
Getting the folowing when choosing the "Split Diff: Enable" option after upgrading to Atom 1.11 on Mac:

```
/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:70

TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:70:13)
    at CompositeDisposable.module.exports.CompositeDisposable.add (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:43:11)
    at new CompositeDisposable (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:19:14)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:265:29
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at PaneContainer.module.exports.PaneContainer.addedPaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:388:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:372:28
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at Pane.module.exports.Pane.addItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:491:20)
    at Object.module.exports.SplitDiff._getVisibleEditors (/Users/dunnga/.atom/packages/split-diff/lib/split-diff.coffee:339:17)
    at Object.module.exports.SplitDiff.diffPanes (/Users/dunnga/.atom/packages/split-diff/lib/split-diff.coffee:183:16)
    at atom-workspace.subscriptions.add.atom.commands.add.split-diff:enable (/Users/dunnga/.atom/packages/split-diff/lib/split-diff.coffee:32:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:183:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:114:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

**Comment by Proteo** [comment]
Same over here, on 1.11.1.

**Comment by azuruce** [comment]
same here

**Comment by mupchrch** [comment]
Appears to be a problem with trying to add a new text editor to a pane. I have filed https://github.com/atom/atom/issues/12963 as a result.

I don't know of any work around at the moment, but I'll keep looking. Sorry about this everyone 😞

**Comment by niij** [comment]
same here. Atom 1.11.1

**Comment by g7morris** [comment]
same issue sadly but can also report two window workaround also doesn't work. Same error.

Atom 1.11.1 on Mac 10.11
Split-diff 1.0.5

**Comment by mupchrch** [comment]
@g7morris I think I may have been unclear in my wording sorry, I'll edit the work around text in my previous comment.

You need to have two tabs side by side within one instance of Atom. So right click a tab or the tab bar, choose Split right, and put the text you want to diff with in a new tab on the right.

**Comment by mupchrch** [comment]
Issue is fixed in v1.11.2
